From 1daa615044c83343fa86221839be2ad551c74d55 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Wed, 14 May 2025 01:12:29 -0700
Subject: [PATCH] fixed test_neighbors import

---
 tests/test_nearest_neighbors.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_nearest_neighbors.py b/tests/test_nearest_neighbors.py
index 641098a..1122ee4 100644
--- a/tests/test_nearest_neighbors.py
+++ b/tests/test_nearest_neighbors.py
@@ -14,7 +14,7 @@ from sklearn.utils import check_random_state
 
 from openTSNE import nearest_neighbors
 from openTSNE.utils import is_package_installed
-from .test_tsne import check_mock_called_with_kwargs
+from test_tsne import check_mock_called_with_kwargs
 
 
 class KNNIndexTestMixin:
-- 
2.39.3 (Apple Git-146)

